src/lib/components/batch-operations/details.svelte

```diff
@@ -1,23 +1,24 @@
 <script lang="ts">
-  import { translate } from '$lib/i18n/translate';
+  import { createTranslate, translate } from '$lib/i18n/translate';
   import { relativeTime, timeFormat } from '$lib/stores/time-format';
   import type { BatchOperation } from '$lib/types/batch';
   import { formatDate } from '$lib/utilities/format-date';
 
   export let operation: BatchOperation;
+  const t = createTranslate('batch');
```

Comment:
Is this really more ergonomic than what it's replacing? I'd love to see something simple like `translate('batch.details')`.

Reply:
I think it's slightly more ergonomic when translating a bunch of strings in the same namespace, however it's probably not going to be realized until we add new pages. I like the idea of passing in a single string like `"<namespace>.<key>"` though.